.gitignore

```diff
@@ -15,3 +15,6 @@ target/
 ext-lib-src/
 .ensime
 .ensime_cache/
+moquette_store.mapdb
+moquette_store.mapdb.p
+moquette_store.mapdb.t
```

Comment:
Shouldn't they go into `target/` if they are generated during tests? Who generates them? Weird. (Not saying it is up to you to fix)